apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -0,0 +1,144 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function InstructorCourseAdminSettings({
+  resLocals,
+  needToSync,
+  noInfo,
+}: {
+  resLocals: Record<string, any>;
+  needToSync: boolean;
+  noInfo: boolean;
+}) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", { ...resLocals })}
+        <style>
+          .popover {
+            max-width: 50%;
+          }
+        </style>
```

Comment:
Are popovers actually used on this page? If not, let's remove this. We can also remove the `<script>` in the body that configures popovers.

Reply:
They are not used. Removed this `<style>` and the `<script>` related to popovers in the body.